tests/Functional/GetRateTest.php

```diff
@@ -33,16 +36,12 @@ public function testGetAllRateRecords()
         $rates = FixtureFactory::createPorter()->import(new ImportSpecification(new GetAllRate($this->apiKey)))
             ->findFirstCollection();
```

Comment:
`findFirstCollection()` should only be used when you need to get metadata. Call it later when testing `getBase()`, but do not call it here, before iteration has been tested.

Reply:
Can you show some example approach for this?

Thanks.

Reply:
Just call `findFirstCollection()` later, after you've done all the other assertions.